### PR TITLE
The preprocess_directory file was modified to include .tif files.

### PR DESCRIPTION
**Pull request by jraval**
# Description

The directory file contains a function that adds files with jpg, png and tif extensions to a directory file.  The functions was modified to include this.  The purpose of creating this file and then to modify this function is so that we have a central location for preprocessed files after they are run through the model.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: data_management/autopreprocess_testing/preprocess_directory.py [hunk 1]
```diff
@@ -19,12 +19,15 @@
 target_dir = os.getcwd() + '/processed_test_images/'
 
 # Get list of all image paths
-img_files = glob.glob(source_dir + "/**/*.jpg", recursive=True)
+jpg_files = glob.glob(source_dir + "/**/*.jpg", recursive=True)
+png_files = glob.glob(source_dir + "/**/*.png", recursive=True)
+img_files = jpg_files + png_files
+tif_files = glob.glob(source_dir + "/**/*.tif", recursive=True)
```
Comment (tinomen) [inline data_management/autopreprocess_testing/preprocess_directory.py:25]: move the tif_files line above the img_files
then add the tif_files to the img_files concat

**Review by tinomen — CHANGES_REQUESTED** [review]
Nice small PR

File: data_management/autopreprocess_testing/preprocess_directory.py (hunk 1 above)
Comment (nlittlepoole) [reply data_management/autopreprocess_testing/preprocess_directory.py:25]: @jraval can you confirm that the preprocessing steps after this work with a `.tif` file? I've never tried it so just want to make sure.

**Comment by jacob-torres** [comment]
Good idea. Just be sure to change the doc string in the preprocessing_pipeline file to reflect .tif files.

**Comment by jraval** [comment]
Restarted pull request, I needed to start a new branch so I decided to redo this pull request.

**Pull request closed without merging**